stargaze/assetlist.json

```diff
@@ -899,6 +899,32 @@
           }
         }
       ]
+    },
+    {
+      "description": "HOOD",
+      "denom_units": [
+        {
+          "denom": "factory/stars1rru5m2wh3fylpheqh8h5g968jwhe7rctkfm7u0gwk7ka4vx3q5tqukjl4z/hood",
+          "exponent": 0
+        },
+        {
+          "denom": "HOOD",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/stars1rru5m2wh3fylpheqh8h5g968jwhe7rctkfm7u0gwk7ka4vx3q5tqukjl4z/hood",
+      "name": "HOOD",
+      "display": "HOOD",
+      "symbol": "HOOD",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stargaze/images/hood.png"
```

Comment:
Please use an image <250kb (this one's 560). Also, p[lease try to remove the extra padding in the image.